### ISSUE TITLE
Face Anti-spoofing detection

### ISSUE THREAD

**Issue by coding-geek21** [akshitagupta15june/Face-X#443]
I'm a GSSoC'21 participant i would like to work on this issue

**Comment by akshitagupta15june** [comment]
Can you please what this feature will actually do?

**Comment by coding-geek21** [comment]
> Can you please what this feature will actually do?

Actually this is for detecting fake faces like someone trying to cheat by showing an image of a face instead of real face

**Comment by coding-geek21** [comment]
> > Can you please what this feature will actually do?
> 
> Actually this is for detecting fake faces like someone trying to cheat by showing an image of a face instead of real face
In front of camera

**Comment by akshitagupta15june** [comment]
Ok you can add this feature inside awesome face operations folder

**Comment by coding-geek21** [comment]
Thank you for assigning me :)

**Comment by akshitagupta15june** [comment]
are you still working on this issue?

**Comment by coding-geek21** [comment]
Yes @akshitagupta15june I'm having exams now so I'm not able to complete it , I will be completing within 1 week

**Comment by akshitagupta15june** [comment]
No problem

**Comment by akshitagupta15june** [comment]
are u working on this?

**Comment by manisha-27** [comment]
Hey, can I work on this issue.
